### PR TITLE
Update migration.md

### PR DESCRIPTION
**Pull request by LouisAUTHIE**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | link is broken 
| Fixed ticket? | n/a

### CODE REVIEW

File: basics/keeping-up-to-date/migration.md [hunk 1]
```diff
@@ -636,4 +636,3 @@ automatic, because:
 Useful links:
 
 -   [Learning how-to use the webservice API](https://doc.prestashop.com/display/PS16/Using+the+PrestaShop+Web+Service)
--   [Getting a list of all available ressources, ready for import/export](https://doc.prestashop.com/display/PS16/Web+service+one-page+documentation#Webserviceone-pagedocumentation-Availableresources)
```
Comment (MeKeyCool) [inline basics/keeping-up-to-date/migration.md:639]: I don't understand, this link works for me :thinking: 
Maybe we should update it to [List of all available ressources](https://devdocs.prestashop.com/8/webservice/resources/) ?

**Review by MeKeyCool — CHANGES_REQUESTED** [review]

File: basics/keeping-up-to-date/migration.md [hunk 2]
```diff
@@ -636,4 +636,3 @@ automatic, because:
 Useful links:
 
 -   [Learning how-to use the webservice API](https://doc.prestashop.com/display/PS16/Using+the+PrestaShop+Web+Service)
```
Comment (kpodemski) [inline basics/keeping-up-to-date/migration.md]:
```suggestion
-   [Learning how to use the webservice API]([web-service])

[web-service]: {{< ref "1.7/webservice" >}}
```

File: basics/keeping-up-to-date/migration.md [hunk 3]
```diff
@@ -635,5 +635,6 @@ automatic, because:
 
 Useful links:
 
--   [Learning how-to use the webservice API](https://doc.prestashop.com/display/PS16/Using+the+PrestaShop+Web+Service)
--   [Getting a list of all available ressources, ready for import/export](https://doc.prestashop.com/display/PS16/Web+service+one-page+documentation#Webserviceone-pagedocumentation-Availableresources)
+-   [Learning how to use the webservice API]([web-service])
```
Comment (kpodemski) [inline basics/keeping-up-to-date/migration.md]:
```suggestion
-   [Learning how to use the webservice API][web-service]
```

File: basics/keeping-up-to-date/migration.md [hunk 4]
```diff
@@ -635,5 +635,6 @@ automatic, because:
 
 Useful links:
 
--   [Learning how-to use the webservice API](https://doc.prestashop.com/display/PS16/Using+the+PrestaShop+Web+Service)
--   [Getting a list of all available ressources, ready for import/export](https://doc.prestashop.com/display/PS16/Web+service+one-page+documentation#Webserviceone-pagedocumentation-Availableresources)
+-   [Learning how to use the webservice API][web-service]
+
+[web-service]: {{< ref "1.7/webservice" >}}
```
Comment (kpodemski) [inline basics/keeping-up-to-date/migration.md]:
```suggestion
[web-service]: {{< ref "8/webservice" >}}
```

File: basics/keeping-up-to-date/migration.md [hunk 5]
```diff
@@ -635,5 +635,6 @@ automatic, because:
 
 Useful links:
 
--   [Learning how-to use the webservice API](https://doc.prestashop.com/display/PS16/Using+the+PrestaShop+Web+Service)
--   [Getting a list of all available ressources, ready for import/export](https://doc.prestashop.com/display/PS16/Web+service+one-page+documentation#Webserviceone-pagedocumentation-Availableresources)
+-   [Learning how to use the webservice API][web-service]
```
Comment (kpodemski) [inline basics/keeping-up-to-date/migration.md]:
```suggestion
-   [How to use the webservice API][web-service]
```

**Review by kpodemski — APPROVED** [review]

**Pull request merged**